Makefile

```diff
@@ -37,13 +37,22 @@ test-unit:
 # Run the code generation tasks.
 # Example:
 #    make update
-update: install-go-bindata
+update: install-go-bindata install-go-deps
 	hack/update-codegen.sh
 	hack/update-generated-bindata.sh
 
 install-go-bindata:
 	hack/install-go-bindata.sh
 
+install-go-deps:
+	go mod tidy
```

Comment:
Should this directive set the `GO111MODULE` environment variable before running `go mod ..`?

This is what I get when I run `make install-go-deps` on my machine:
```
go mod tidy
go: modules disabled inside GOPATH/src by GO111MODULE=auto; see 'go help modules'
make: *** [Makefile:48: install-go-deps] Error 1
```

Reply:
likely, I think we can bring much from https://github.com/openshift/cluster-monitoring-operator/blob/master/Makefile#L76 :) as they already successfully moved to go modules

Reply:
yes, good catch! I forgot to set this explicitly; had it disabled globally.